webpack/components/extensions/HostDetails/Tabs/RemoteExecutionActions.js

```diff
@@ -209,6 +209,16 @@ export const removePackage = ({ hostname, packageName }) => post({
   errorToast,
 });
 
+// Used by packages wizard
+export const removePackages = ({ hostname, search, descriptionFormat }) => post({
+  type: API_OPERATIONS.POST,
+  key: REX_JOB_INVOCATIONS_KEY,
+  url: foremanApi.getApiUrl('/job_invocations'),
+  params: katelloPackageRemoveParams({ hostname, search, descriptionFormat }),
+  handleSuccess: showRexToast,
+  errorToast,
+});
```

Comment:
This still is not working for me. Double check https://github.com/Katello/katello/pull/11070/files

Instead of `katelloPackageRemoveParams` I think you need to add back `katelloPackagesRemoveParams` and then use that.

Reply:
Added back